### DISCUSSION TITLE
Can not install gatsby-remark-autolink-headers plugin

### DISCUSSION THREAD

**Discussion by kmerimi** [gatsbyjs/gatsby#29991] [Help]
## Description

When I install the plugin with the following command "npm install gatsby-remark-autolink-headers" I had the error:

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: gatsby-starter-default@0.1.0
npm ERR! Found: gatsby@2.32.9
npm ERR! node_modules/gatsby
npm ERR!   gatsby@"^2.26.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer gatsby@"^3.0.0-next.0" from gatsby-remark-autolink-headers@3.0.0
npm ERR! node_modules/gatsby-remark-autolink-headers
npm ERR!   gatsby-remark-autolink-headers@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/vegeta/.npm/eresolve-report.txt for a full report.

### Environment
Mac

**Comment by LekoArts** [comment]
Hi!

It seems you're still using `gatsby` v2 and try to install the latest `gatsby-remark-autolink-headers` plugin that has a peerDependency on `gatsby` v3.

As the error suggests you can try this instead:

```shell
npm install gatsby-remark-autolink-headers --legacy-peer-deps
```

**Reply by kmerimi** [reply]
Thanks You I will upgrade gatsby to V3
